pandera/io.py

```diff
@@ -132,9 +133,17 @@ def _deserialize_component_stats(serialized_component_stats):
         "pandas_dtype": pandas_dtype,
         "nullable": serialized_component_stats["nullable"],
         "checks": checks,
-        **(
-            {} if "name" not in serialized_component_stats else
-            {"name": serialized_component_stats["name"]}
+        **ChainMap(
```

Comment:
consider something like this here:

```python
**{
    key: serialized_component_stats.get(key) for key in
    ["name", "allow_duplicates", "coerce", "required", "regex"]
    if key in serialized_component_stats
}
```

Reply:
Great idea - that's a lot cleaner! I'll make the change and address unit testing as well.